website/project/metadata/real-world-evidence.json

```diff
@@ -0,0 +1,233 @@
+               "help":"01/15/2010 to 09/22/2019 (can include a future date as well)",
+               "required":true
+            }
+         ]
+      },
+      {
+         "id":"page2",
+         "title":"Summary Design and Summary Specifications",
+         "questions":[
+            {
+               "qid":"q6",
+               "type":"choose",
+               "format":"singleselect",
+               "title":"Study Design",
+               "description":"Please choose one",
+               "options":[
+                  "Cohort",
+                  "Case-Control",
+                  "Self-controlled (e.g. pre-post)",
+                  "Cross-sectional",
+                  "Other (describe in text box below)"
+               ],
+               "help":"Cohort designs - A study design where one or more samples (called cohorts) are identified and patients categorized by treatment or exposure groups and followed for a defined period of time with subsequent status evaluations of the outcome(s) of interest. Cohort studies are conducted to determine the magnitude of treatment associations with the outcome. As the study is conducted, outcomes from participants in each cohort are measured and relationships with specific characteristics are determined. Within cohorts, one can apply sampling designs that oversample patient according to certain study parameters to increase study efficiency: Case-control sampling identifies patients according to their outcome status; case-cohort sampling identifies patients according to exposure status and pre-exposure patient characteristics; and two-stage sampling which does both. The efficiency gain with cohort sampling approaches is most relevant if some information critical to the study is difficult or costly to acquire. The family of self-controlled designs use an individual’s experience at different times as their own control. We differentiate self-controlled designs by how patients are selected into the study, or the indexing event: the case-crossover design is indexed by the study outcome, while the self-controlled case series is typically exposure-indexed.\nQuasi-experimental: research that resembles experimental research but is not true experimental research. Often but not always a cohort design, it also involves a condition or analytical technique that proxies randomization of treatment assignment to some degree but is not true randomization.",
+               "required":true
+            },
+            {
+               "qid":"q7",
+               "type":"string",
+               "format":"textarea",
+               "title":"Other",
+               "required":false
+            },
+            {
+               "qid":"q8",
+               "type":"string",
+               "format":"textarea",
+               "title":"Study population",
+               "description":"Please choose one the population under evaluation in the study",
```

Comment:
This was wrong in the spreadsheet. It should be, "The population under evaluation in the study"